src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -1296,29 +1293,34 @@ By default, we include the following package managers in each runtime.
 {% /table %}
 
 # Using Appwrite in a function {% #using-appwrite %} 
-Appwrite can be used in your functions by adding the relevant SDK to your function's dependencies. 
-Authenticating with Appwrite is done via an API key or a JWT token. 
-API keys must be generated and exported as an [environment variable](/docs/advanced/self-hosting/environment-variables).
+Appwrite can be used in your functions by adding the relevant SDK to your function's dependencies.
+Integrate with Appwrite using its [authentication](/docs/products/auth/jwt), which is done via a JWT token or a dynamic API key. 
```

Comment:
> Integrate with Appwrite using its [authentication](/docs/products/auth/jwt)

This doesn't really sound good. Also, you mention authentication, but link to JWT.